src/app/EventManagement.cpp

```diff
@@ -580,6 +585,9 @@ CHIP_ERROR EventManagement::CopyEvent(const TLVReader & aReader, TLVWriter & aWr
 
 static bool IsInterestedEventPaths(EventLoadOutContext * eventLoadOutContext, const EventEnvelopeContext & event)
 {
+    ChipLogError(EventLogging, "current fabric index is %u", eventLoadOutContext->mFabricIndex);
+    ;
```

Comment:
Nit: extra `;`

Reply:
this is one debug log, would remove it soon.

Reply:
Done, thanks, It is ready for formal review.